tools/accuracy_checker/accuracy_checker/preprocessor/one_hot_encoding.py

```diff
@@ -0,0 +1,80 @@
+"""
+Copyright (c) 2020 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import numpy as np
+from ..config import NumberField, BoolField
+from .preprocessor import Preprocessor
+
+
+class OneHotEncoding(Preprocessor):
+    __provider__ = 'one_hot_encoding'
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'value': NumberField(
+                optional=True,
+                value_type=int,
+                default=1,
+                description="Number for label encoding. Integer which used to indicate label"
+            ),
+            'number_of_classes': NumberField(
+                optional=True,
+                value_type=int,
+                default=1,
+                description="Number of encoding classes"
+            ),
+            'axis': NumberField(
+                optional=True,
+                value_type=int,
+                default=1,
+                description="Axis for encoding"
+            ),
+            'base': NumberField(
+                optional=True,
+                value_type=int,
+                default=0,
+                description="Number that specifies the values of the other classes at one hot label map"
+            )
+        })
+        return parameters
+
+    def configure(self):
+        self.value = self.config.get('value')
```

Comment:
please use get_value_from_config instead, it handles default values and type casting